lib/utils/lazy.ts

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 import * as BalenaSdk from 'balena-sdk';
+import chalk from 'chalk';
```

Comment:
Being the 3rd PR, you know what my comment here would be... :-)

Reply:
I'll fix it the moment I can :P